raiden/utils/__init__.py

```diff
@@ -289,3 +289,10 @@ def merge_dict(to_update: dict, other_dict: dict):
             merge_dict(to_update[key], value)
         else:
             to_update[key] = value
+
+
+def optional_address_to_string(address: typing.Address = None) -> typing.Optional[str]:
```

Comment:
the `optional_` prefix is used to indicate the type of return value which IMO is not required. `address_to_string` should suffice.

Reply:
I mostly wanted to indicate that the function can handle optional inputs, otherwise `to_checksum_address` could be used directly.